sshd-core/src/main/java/org/apache/sshd/client/session/ClientUserAuthService.java

```diff
@@ -124,7 +128,15 @@ public Map<String, Object> getProperties() {
 
     @Override
     public void start() {
-        // ignored
+        Runnable initial = null;
```

Comment:
No need to initialize *initial* to *null* since its value is set unconditionally inside the *synchronized* block.

Reply:
Done.